base/views.py

```diff
@@ -1193,6 +1194,46 @@ def mail_server_conf(request):
     )
 
 
+@login_required
+@permission_required("base.view_dynamicemailconfiguration")
+def mail_server_test_email(request):
+    instance_id = request.GET.get("instance_id")
+    form = DynamicMailTestForm()
+    if request.method == "POST":
+        form = DynamicMailTestForm(request.POST)
+        if form.is_valid():
+            email_to = form["to_email"]
```

Comment:
Hi @gehasia ,
On testing we have faced issue in this section. The mail was sent correctly, but an error message was popping up.
It should be 
```
email_to = form.data["to_email"]
```

Reply:
My bad, in fact it hase already been corrected but included in another PR i will push today. I will add it to this PR later todauy as this belong to this PR more than the other !

Reply:
Okay 😅

Reply:
Ok i have added the commit to fix this !